File: rc-application/src/main/java/com/transkript/reportcard/business/service/StudentApplicationTrialService.java
package com.transkript.reportcard.business.service;

import com.transkript.reportcard.data.entity.relation.StudentApplicationTrial;

import java.util.List;

public interface StudentApplicationTrialService {
    StudentApplicationTrial getEntity(Long satId);

    List<StudentApplicationTrial> getEntitiesByYear(Long yearId);
}

File: rc-application/src/main/java/com/transkript/reportcard/business/service/StudentApplicationTrialService.kt
package com.transkript.reportcard.business.service

import com.transkript.reportcard.data.entity.relation.StudentApplicationTrial

interface StudentApplicationTrialService {
    fun getEntity(satId: Long?): StudentApplicationTrial?
    fun getEntitiesByYear(yearId: Long?): List<StudentApplicationTrial?>?
}
